client/my-sites/site-settings/seo-settings/services.js

```diff
@@ -0,0 +1,44 @@
+// Add new Site Verification services to this file.
+// Requires the API to be updated in Jetpack in order to work.
+
+/**
+ * Returns the Site Verification services.
+ *
+ * @param  {Function}  translate  Translate needed for the name.
+ * @returns {object}              Site Verification services data.
```

Comment:
Looking good! Tests well for me on sites with & without the related JP 9.8 code. One small change here:

```suggestion
 * @returns {Array}              Site Verification services data.
```

Reply:
Nice spot, thanks! Fixed. :)